docs/_includes/Modules.md

```diff
@@ -15,12 +15,12 @@ interface.
 ## 4.1. Declaring New Modules
 ````grammar
 ModuleDefinition_ = "module" { Attribute } ModuleName
-        [ [  "exclusively" ] "refines" ModuleQualifiedId ]
+        [ [ "exclusively" ] "refines" QualifiedModuleName ]
```

Comment:
```suggestion
        [ "refines" QualifiedModuleName ]
```

The `exclusively` was dropped several years ago.

Reply:
OK